systemuser.sh

```diff
@@ -38,12 +38,16 @@ mkdir -p $KERNEL_DIR
 export JUPYTER_RUNTIME_DIR=$JPY_LOCAL_DIR/share/jupyter/runtime
 export IPYTHONDIR=$SCRATCH_HOME/.ipython
 mkdir -p $IPYTHONDIR
+export PROFILEPATH=$IPYTHONDIR/profile_default
```

Comment:
Do we really need an ipython profile or we can put this configuration in $JUPYTER_CONFIG_DIR/jupyter_notebook_config.py? Or it has to be an ipython configuration thing?

Reply:
Yes, this is an iPy configuration. So it needs to be here.